callbacks/control_bar.py

```diff
@@ -0,0 +1,74 @@
+from dash import Input, Output, State, callback, Patch, MATCH, ALL, ctx
+import dash_mantine_components as dmc
+import json
+
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Input("annotation-opacity", "value"),
+    prevent_initial_call=True,
+)
+def annotation_opacity(opacity_value):
+    patched_figure = Patch()
+    patched_figure["layout"]["newshape"]["opacity"] = opacity_value
+    return patched_figure, 3
```

Comment:
What's the `3` returned for here? I'm getting an error when updating the annotation opacity:

```
Invalid argument `figure` passed into Graph with ID "image-viewer".
Expected `object`.
Was supplied type `array`.
Value provided: 
[
  {
    "__dash_patch_update": "__dash_patch_update",
    "operations": [
      {
        "operation": "Assign",
        "location": [
          "layout",
          "newshape",
          "opacity"
        ],
        "params": {
          "value": 0.4
        }
      }
    ]
  },
  3
]
```

Reply:
Must have been typo, good catch